services/site/tests/components/challenge/Editors.spec.tsx

```diff
@@ -25,21 +25,21 @@ const handleReset = jest.fn();
 
 describe("Editors", () => {
   it("renders the wrapper element with classes", () => {
-    const { container } = render(<Editors className={mockClassName} editors={[htmlEditorConfig]} onReset={handleReset} />);
+    const { container } = render(<Editors className={mockClassName} editors={[htmlEditorConfig]} onReset={handleReset} autoSave={true} />);
```

Comment:
Why do you add autoSave when it's optional and you don't test for it? You could just omit it to keep the tests short and simple